org.deluge_torrent.deluge.yaml

```diff
@@ -0,0 +1,437 @@
+app-id: org.deluge_torrent.deluge
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: deluge-gtk
+#rename-icon: deluge
+#copy-icon: true
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=xdg-download
+
+modules:
```

Comment:
You should add a global or per-module `cleanup` array to remove development headers and static libraries.

Reply:
added cleanup for boost and libtorrent